src/csharp/Microsoft.Spark/Utils/CommandSerDe.cs

```diff
@@ -216,11 +216,51 @@ private sealed class UdfWrapperData
             }
         }
 
-        internal static T Deserialize<T>(
+        internal static object DeserializeArrowOrDataFrameUdf(
             Stream stream,
             out SerializedMode serializerMode,
             out SerializedMode deserializerMode,
-            out string runMode) where T : Delegate
+            out string runMode)
+        {
+            var udfWrapperData = GetUdfWrapperDataFromStream(stream, out serializerMode, out deserializerMode, out runMode);
+
+            var nodeIndex = 0;
+            var udfIndex = 0;
+            UdfWrapperNode node = udfWrapperData.UdfWrapperNodes[nodeIndex];
+            var nodeType = Type.GetType(node.TypeName);
+            Delegate udf = null;
+            if (nodeType == typeof(DataFrameGroupedMapUdfWrapper))
+            {
+                udf = (DataFrameGroupedMapWorkerFunction.ExecuteDelegate)DeserializeUdfs<DataFrameGroupedMapWorkerFunction.ExecuteDelegate>(udfWrapperData, ref nodeIndex, ref udfIndex);
```

Comment:
Can you help making the repo consistent with max char limit by following https://github.com/dotnet/spark/blob/master/docs/coding-guidelines/csharp-coding-style.md? Thanks!